src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardTokens.tsx

```diff
@@ -182,6 +187,23 @@ const IfoCardTokens: React.FC<IfoCardTokensProps> = ({
       )
     }
 
+    const ifov31Msg =
+      ifo.version === 3.1 && poolId === PoolIds.poolBasic && criterias?.length > 0 ? (
+        <Box mt="16px">
+          {!isEligible && (
+            <Message mx="24px" mb="24px" p="8px" variant="danger">
+              <MessageText display="inline">Meet any one of the following requirements to be eligible.</MessageText>
```

Comment:
Missing translation

Reply:
Yeah. Thanks. It's just a draft. I will add translation later.